pkgs/os-specific/darwin/apple-source-releases/default.nix

```diff
@@ -238,7 +238,7 @@ let
 
 in
 
-developerToolsPackages_11_3_1 // macosPackages_11_0_1 // {
+developerToolsPackages_11_3_1 // macosPackages_11_0_1 // rec {
```

Comment:
Relying on `rec` prevents you from being able to properly propagate overrides

Reply:
Okay. I can update it to call `applePackage` like the other derivations. It shouldn’t use `pkgs.darwin.configd` because that will get replaced by the SDK package on `aarch64-darwin`, which is the problem that’s being worked around. Should I add a comment explaining why it’s not following the pattern of the other headers-only variants?

Also, would it be better to default `Security` to `null` in the call or pass it in even though it’s not going to be used in the headers-only derivation?